deployment/urunc-deploy/urunc-deploy/base/urunc-deploy.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: urunc-deploy
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: urunc-deploy
```

Comment:
It seems that in all other files, we use the `kubelete-urunc-deploy` name. Is there a reason we use a different name here? If not would it be possible to use the same name everywhere?

Reply:
We use the name `kubelet-urunc-cleanup` for the cleanup deployment. I think we could also change `urunc-deploy` to `kubelet-urunc-deploy` , if you think it is better that way.